modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -49,11 +49,11 @@ namespace cv {
  *
  * @sa GCompiled
  */
-class GAPI_EXPORTS GStreamingCompiled
+class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
```

Comment:
@TolyaTalamanov ☝️

Reply:
I see, but if this object is returned from function it should be `SIMPLE` 
`Net` - https://github.com/opencv/opencv/blob/master/modules/dnn/include/opencv2/dnn/dnn.hpp#L390
`function` https://github.com/opencv/opencv/blob/master/modules/dnn/include/opencv2/dnn/dnn.hpp#L759

Because for `CV_EXPORTS_W` wrapper generates code for `Ptr<Object>` but function return `Object` by value, so it means in case `Object` is returned from function need to use `CV_EXPORTS_W_SIMPLE`, doesn't it ?

Reply:
So you are right, need to avoid it for objects which is not `pimpl`, but in case `GCompiledStreaming` it works fine. Opened a ticket